### ISSUE TITLE
Weight initialization Error on using pretrained model in pytorch

### ISSUE THREAD

**Issue by adiv5** [EmilyAlsentzer/clinicalBERT#16]
I am getting this error 
```
Weights of BertForMultiLable not initialized from pretrained model: ['classifier.weight', 'classifier.bias']
Weights from pretrained model not used in BertForMultiLable: ['cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.decoder.weight', 'cls.seq_relationship.weight', 'cls.seq_relationship.bias']
```
while using the provided pretrained biobert from here in this [repo](https://github.com/lonePatient/Bert-Multi-Label-Text-Classification).
here is the [issue](https://github.com/lonePatient/Bert-Multi-Label-Text-Classification/issues/36) for reference.

Is there some issue in the model?

**Comment by EmilyAlsentzer** [comment]
Hi Aditya,

Can you clarify what you are doing that gets this error? Are you trying to use clinical BERT model in the instructions provided by the Bert-Multi-Label-Text-Classification repo? Is the issue that it works when you use BERT but not clinical BERT or do you get the same error regardless of which BERT model you use?

**Comment by EmilyAlsentzer** [comment]
Closing this issue due to lack of activity. Feel free to reopen and provide more details.

**Comment by YojanaGadiya** [comment]
Hey! I get the same error when I am trying use load the clinicalbert model as a BERT model in BertforTokenClassification with transformers. Do you know the reasons for the same?
